core/src/com/agateau/pixelwheels/gameinput/SidesTouchInputHandler.java

```diff
@@ -61,11 +62,10 @@ public Array<GameInputHandler> getAllHandlers() {
     public GameInput getGameInput() {
         mInput.braking = isBraking();
         mInput.accelerating = !mInput.braking;
-        if (!mInput.braking) {
```

Comment:
I think you need to keep the `if (!mInput.braking) {` here: in this input mode, pressing both directions is used to drive backward.

Reply:
this meant to propagate the player action to `DigitalSteering` so the trickery logic can be done there. I've elaborated the description by added comments there. I admit straight-braking is sacrificed in favor of fast-reverse-at-a-direction by doing the logic there. Please review the updated comments there and give further advice.